joomla/joomla-cms
# [4.0] Prepare Database for checked_out in workflow tables

This pr adds the fields checked_out and checked_out_time to the workflow tables.
If this is done, the views of the workflow component can use the checked_out functions as other components. 

### Testing Instructions
Install a new application and inspect tables: workflows, workflow_stages and workflow_transitions.
Everything should work as before. 

NOTE: I could test only the mysql database Would be great if I get help with testing the postgres part.
